api/src/main/resources/workbench.yaml

```diff
@@ -91,6 +91,32 @@ paths:
           schema:
             $ref: "#/definitions/UsernameTakenResponse"
 
+  /api/v1/create-account:
+    post:
+      tags:
+        - profile
+      description: Creates an account in the researchallofus.org domain.
+      operationId: createAccount
+      security: []
```

Comment:
So... once we do this, I believe anybody on the internet is going to be able to call this endpoint and create a GSuite account under this domain.

The odds of somebody actually wanting to do this are probably pretty low, and maybe we can live with this. But we may want to consider locking this down in some way. (I'm not sure how.)

At some point we will probably want Captcha integration or something to ensure bots don't spam us with fake accounts. That might tie into this somehow.

Reply:
Per our discussion, etc. we need some way of whitelisting AofU devs for creating accounts for now.

One possible solution: have a shared secret password only we know that you have to use when creating accounts.

Reply:
I chatted with @jmandel about this. He thinks eventually we will need invite codes we can generate and send out to external parties; being able to sign up for an account will be gated on having an invite code.

There's a bunch of work involved in setting that up, so for now a shared secret password is probably fine.

Reply:
I went with a secret invitation key. PTAL @danqrodney @jmandel